### PR TITLE
fix: public squad creation revamp

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Revamp squad overview page (small header changes and spacing concerns)
- Revamp new squad page, basically revamp the selection (which is still disabled)
- Had to adjust our radio buttons to accept singular disabled state

Screens of new:
Screenshot 2023-10-06 at 16 10 09
Screenshot 2023-10-06 at 16 10 02

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-1814 #done

### CODE REVIEW

File: packages/shared/src/components/squads/Details.tsx [hunk 1]
```diff
@@ -215,26 +243,12 @@ export function SquadDetails({
...
-                  tag: 'a',
-                  target: '_blank',
-                  href: squadsPublicWaitlist,
-                }}
+            <div>
+              <Radio
+                name="squadType"
+                options={squadTypeOptions}
+                value="private"
+                onChange={() => {}}
```
Comment (rebelchris) [inline packages/shared/src/components/squads/Details.tsx:251]: Intended as it doesn't do anything 😅

File: packages/shared/src/components/squads/SquadsDirectoryHeader.tsx [hunk 2]
```diff
@@ -1,62 +1,50 @@
...
         }}
       >
-        <SourceBetaIcon
-          width="auto"
-          height="3rem"
-          className="mb-4 text-white translate-x-[1.125rem]"
-        />
-        <div className="flex justify-center items-center mb-3 text-white typo-large-title">
-          Introducing Squads
+        <div className="flex relative z-1 flex-col items-center">
```
Comment (rebelchris) [inline packages/shared/src/components/squads/SquadsDirectoryHeader.tsx:19]: Basically wrapped for z-index

File: packages/shared/src/components/squads/Details.tsx [hunk 3]
```diff
@@ -12,14 +12,12 @@ import CameraIcon from '../icons/Camera';
...
 import { checkExistingHandle, SquadForm } from '../../graphql/squads';
-import { capitalize } from '../../lib/strings';
+import { anchorDefaultRel, capitalize } from '../../lib/strings';
 import { IconSize } from '../Icon';
 import SquadIcon from '../icons/Squad';
 import { SourceMemberRole } from '../../graphql/sources';
 import { Radio } from '../fields/Radio';
-import BetaBadge from '../../svg/BetaBadge';
 import { squadsPublicWaitlist } from '../../lib/constants';
-import { SquadTypeCard } from './SquadTypeCard';
```
Comment (capJavert) [inline packages/shared/src/components/squads/Details.tsx:22]: We can also delete the file

**Review by capJavert — APPROVED** [review]
Like how it looks 👍 LGTM!

File: packages/shared/src/components/squads/Details.tsx [hunk 4]
```diff
@@ -58,6 +56,37 @@ const memberRoleOptions = [
   },
 ];
 
+const squadTypeOptions = [
+  {
+    label: 'Private Squad',
+    value: 'private',
+    afterElement: (
+      <p className="mb-5 ml-10 typo-footnote">
```
[the remark this answers is not in the document]
Comment (omBratteng) [reply packages/shared/src/components/squads/Details.tsx:64]: I've only looked at the attached screenshot, and not checked in browser how it looks

[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/squads/Details.tsx:64]: It's massive, but also massive in the design 😅

[the remark this answers is not in the document]
Comment (omBratteng) [reply packages/shared/src/components/squads/Details.tsx:64]: Yeah, I'm looking at the design. Wonder if design will agree to give it a little lift, a couple of pixels to reduce the gap.

[the remark this answers is not in the document]
Comment (rebelchris) [reply packages/shared/src/components/squads/Details.tsx:64]: You can check in prod now.
Feel free to raise it seperate for design :)

**Pull request merged**